ccd-definition/CaseField/CaseFieldLRspec.json

```diff
@@ -669,5 +697,12 @@
     "Label": "Disclosure of electronic documents",
     "FieldType": "DisclosureOfElectronicDocumentsLRspec",
     "SecurityClassification": "Public"
+  },
```

Comment:
This results in duplicate text on the screen. Please could you check

Reply:
I didn't realize the same text was in 9678 branch, thank you